rita_client/src/operator_update/mod.rs

```diff
@@ -209,6 +209,10 @@ async fn checkin() {
 
     let mut rita_client = settings::get_rita_client();
 
+    //match new_settings.contact_info {
+    //Some(sequence) =>
```

Comment:
here we should take and store the new contact info if it has the higher sequence.